sogintegration.module

```diff
@@ -82,8 +85,9 @@ function sogintegration_saveDirectDebit(&$form) {
  * Create a new SOG user account through the SOG Dashboard API
  */
 function sogintegration_createSogUser(&$form) {
-        require_once DASHBOARD_DIR . '/vendor/autoload.php';
-        require_once(DASHBOARD_DIR . '/app/config.php');
+        // This line of code is complete guesswork based on
+        // https://www.drupal.org/docs/8/modules/http-client-manager/introduction
```

Comment:
We're not using drupal v8, AFAIK, sadly

Reply:
In that case we can just import the library from composer. See [this](https://github.com/laurenzfg/guzzle_vogelnest) for reference.

Reply:
If the upcoming commit / including guzzle doesn't work or is considered bloat, we can use a library function by Drupal 7 instead: [_drupal_http_request_}(https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_http_request/7.x)